quickstart/speech-translation/cpp-windows/helloworld/pch.cpp

```diff
@@ -0,0 +1,13 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
+//
+
+// <code>
+
```

Comment:
we don't need the <code> tag in here? Please only put it where we'll reference the code from the azure-docs page